src/components/create-user.html

```diff
@@ -109,12 +109,24 @@ <h5>Create</h5>
   <script type="module">
     import { Element as PolymerElement } from '@banno/polymer/polymer-element.js';
 
-    class CreateUser extends PolymerElement {
+    // Routing Imports
+    import routeTree from '../routing/routes.js';
+    import Route_Paths from '../routing/paths.js';
+    import Route_Ids from '../routing/id.js';
+    import bindPropertiesFromParentRouteMixin from '../../src/routing/bindPropertiesFromParentRouteMixin.js';
+    const router = require('../../node_modules/@banno/web-component-router');
+    const pageJs = require('page');
+    const RouteMixin = require('@banno/web-component-router/routing-mixin');
+
+    class CreateUser extends bindPropertiesFromParentRouteMixin(RouteMixin(PolymerElement)) {
       static get is() { return 'create-user'; }
 
       static get properties() {
         return {
-          storage: Object,
+          storage: {
+            type: Object,
+            observer: 'check'
```

Comment:
probably nice when deving, but I don't think you'll actually want to merge this.

also, you could do this all in one line:
```suggestion
            observer: (newVal, oldVal) => console.log(newVal)
```